src/App.jsx

```diff
@@ -14,45 +16,85 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+export const App = () => {
+  const [sortedItems, setSortedItems] = useState(goodsFromServer);
+  const [isAlphaSorted, setIsAlphaSorted] = useState(false);
+  const [isLengthSorted, setIsLengthSorted] = useState(false);
+  const [isReversed, setIsReversed] = useState(false);
```

Comment:
you can use only two states:
const [sortBy, setSortBy] = useState('');
const [isReversed, setIsReversed] = useState('');